paymaster-relayer/pyproject.toml

```
@@ -0,0 +1,101 @@
+[project]
+name = "paymaster_relayer"
```

Since this is Sapphire and ROFL-specific, we should name it as such if it's going to be used as an external package.

```suggestion
name = "oasis_paymaster_relayer"
```